src/App.tsx

```diff
@@ -127,77 +193,80 @@ export const App: React.FC = () => {
 
             <button type="button" className="todo__remove" data-cy="TodoDelete">
               ×
-            </button>
+            </button> */}
 
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div data-cy="TodoLoader" className="modal overlay is-active">
+          {/* 'is-active' class puts this modal on top of the todo */}
+          {/* <div data-cy="TodoLoader" className="modal overlay is-active">
               <div className="modal-background has-background-white-ter" />
               <div className="loader" />
             </div>
-          </div>
+          </div> */}
         </section>
 
         {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
+        {todos.length !== 0 && (
+          <footer
+            className={cn('todoapp__footer', { hidden: !todos.length })}
+            data-cy="Footer"
+          >
+            <span className="todo-count" data-cy="TodosCounter">
+              {todosCounter}
+            </span>
 
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
+            {/* Active link should have the 'selected' class */}
+            <nav className="filter" data-cy="Filter">
```

Comment:
It could be helpful to take this filter into separate component. There was nothing about creating such components in the readme file. But just think about it)